src/raw/requests.rs

```diff
@@ -81,19 +83,19 @@ impl KvRequest for kvrpcpb::RawBatchGetRequest {
     const RPC_FN: RpcFnType<Self, Self::RpcResponse> = TikvClient::raw_batch_get_async_opt;
 
     fn make_rpc_request<KvC: KvClient>(&self, keys: Self::KeyData, store: &Store<KvC>) -> Self {
-        let mut req = store.request::<Self>();
+        let mut req = self.request_from_store(store);
         req.set_keys(keys.into_iter().map(Into::into).collect());
         req.set_cf(self.cf.clone());
 
         req
     }
 
-    fn store_stream<PdC: PdClient>(
+    fn store_builder_stream<PdC: PdClient>(
```

Comment:
I think you can leave the name, "store_builder_stream" is a bit long and doesn't add much useful info